build/webpack/webpack.client.config.js

```diff
@@ -18,21 +18,25 @@ module.exports = {
     output: {
         path: path.join(constants.ExtensionRootDir, 'out', 'client_renderer'),
         filename: '[name].js',
-        chunkFilename: `[name].bundle.js`
+        chunkFilename: `[name].bundle.js`,
+        libraryTarget: 'module'
```

Comment:
This is the change we needed webpack 5 for. Import loading now for notebook renderers:
https://github.com/microsoft/vscode-github-issue-notebooks/blob/e47c04e301025f4b024b70abadb48c73e7b13374/webpack.config.js#L72

https://github.com/microsoft/vscode/issues/123540#issuecomment-839067764